### PR TITLE
use the new website design from ElementalCode/elemental#220

### PR DESCRIPTION
**Pull request by mrjacobbloom**
I haven't done any of the logged-in stuff yet 'cause I'm having trouble registering a new account on localhost. But I broke down the HTML into navbar/index/footer/etc. and tried to do the same for the SASS, and I updated the login and signup pages.

### CODE REVIEW

**Comment by mrjacobbloom** [comment]
see ElementalCode/elemental#220

File: static/homepage.scss [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+
+.centered {
+  width:90%;
+  max-width:900px;
+  margin:0 auto;
+}
+
+/* == Logged out user things == */
+// Is it possible to only serve these when the user is logged out?
+#logged-out-header {
```
Comment (matthewr6) [inline static/homepage.scss]: Could we use classes instead of IDs?

**Comment by matthewr6** [comment]
Other than the comment above, it looks good

File: static/homepage.scss (hunk 1 above)
Comment (mrjacobbloom) [inline static/homepage.scss]: Done

**Comment by mrjacobbloom** [comment]
There's still a lot of stuff in style.scss that might not be used--I haven't checked

**Comment by mrjacobbloom** [comment]
Is there anything else you want added before merging this?

**Comment by matthewr6** [comment]
I just forgot to merge it, oops

**Pull request merged**